### ISSUE TITLE
Error on large tables (ORA-06502: PL/SQL: numeric or value error)

### ISSUE THREAD

**Issue by rimblas** [Pretius/apex-nested-reports#7]
Getting the following error with large (more than 32K is my guess) nested tables.

> We're Sorry. An unexpected error has occurred. Please note the following information and contact the help desk:<p/><PRE/><BR/> Application ID: 4000250<BR/>        Page ID: 300<BR/>APEX ERROR CODE: WWV_FLOW_PLUGIN.RUN_PLSQL_ERR<BR/>    ora_sqlcode: -6502<BR/>    ora_sqlerrm: ORA-06502: PL/SQL: numeric or value error
ORA-06512: at line 504
ORA-06502: PL/SQL: numeric or value error</PRE>

The culprit line 494 in the code:
`htp.p( apex_json.get_clob_output );`

I believe htp.p cannot handle such large values without them being split.
However, the fix I recommend is simply to let APEX_JSON output the to the stream like this:

```
  -- apex_json.initialize_clob_output;

  apex_json.open_object;
  apex_json.write( 'data', l_sys_cursor );
  apex_json.open_array('headers');

  for i in 1..l_desc_col_no loop
    apex_json.open_object;
    apex_json.write('COLUMN_NAME', l_desc_col_info(i).col_name);
    apex_json.write('COLUMN_TYPE', getColumnTypeString( l_desc_col_info(i).col_type ) );
    apex_json.close_object;
  end loop;
  
  apex_json.close_array;
  
  apex_json.write( 'x01', l_ajax_column_name, true );
  apex_json.write( 'x02', l_ajax_column_values, true );

  apex_json.close_object;

  -- htp.p( apex_json.get_clob_output );
```

So, remove `apex_json.initialize_clob_output` and the `htp.p` call.

**Comment by lazaa** [comment]
same issue for me

**Comment by clinville2003** [comment]
I also had this same issue.  Errors I got where the nested results would be are
"unexpected end of json input" and "Ajax response could not be parsed as JSON"
Reducing columns or the resulting rows brought the row results back, demonstrating that it was maxing something out.

Commenting out those 2 lines in the plug-in source did the trick.  Thank you @rimblas

**Comment by bostrowski** [comment]
@rimblas, @lazaa, @clinville2003 
Thank you for your feedback. The issue will be fixed in next patch.

**Comment by griz1991** [comment]
This solution is correct. I was able to see only 80 rows and when i added 81th row in my table the "Unexpected end of JSON input" error was thrown. Whit this solution i can see more than 800 rows now.

**Comment by vasukisivalingam** [comment]
Hi 
I am also using same pretius nested report plugin,I too getting below error.
but my sub-report having less than 15 rows and not more than 4 columns.

I tried removing the htp.p as you mentioned above even I am getting error

Could you please help me on this.
![err](https://user-images.githubusercontent.com/45201970/48774401-3167ea00-ecf0-11e8-8518-9b0d8dbb9d0d.png)

**Comment by bostrowski** [comment]
included in v2.0.2. Thanks!

**Comment by rajkogojkovic** [comment]
omg, this solution realy works! Thx @rimblas 
I wonder is there some downside effects with this solution?
